Cargo.toml

```diff
@@ -20,7 +20,7 @@ dump_ivf = ["ivf"]
 
 [dependencies]
 bitstream-io = "0.8"
-clap = { version = "2", optional = true, default-features = false }
+clap = { git = "https://github.com/clap-rs/clap", branch = "v3-master", optional = true, default-features = false }
```

Comment:
Because we don't have a Cargo.lock and v3-master could add breaking changes at any time, I think we'll need to lock this to a particular revision.

Reply:
Would we be fine with locking this to a specific commit hash?

Reply:
Oof and it fails on Windows. I'll have to find a commit that builds on windows.

Reply:
is `v3` developed btw? I started poking the git tree and it is strange.